### PR TITLE
JC-22. Add negative scenario test

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

File: src/test/java/org/jacp/repository/QuestionRepositoryTest.java [hunk 1]
```diff
@@ -33,4 +33,13 @@ void testFindById() {
             assertEquals("Grasshopper", entity.getProblem());
         });
     }
+
+    @Test
+    void testFindByIdNegative() {
+        Optional<QuestionEntity> optionalQuestion = questionRepository.findById(QUESTION_ID);
+        assertTrue(optionalQuestion.isPresent());
```
Comment (pelmegov) [inline src/test/java/org/jacp/repository/QuestionRepositoryTest.java:40]: assertFalse?

Comment (saffchen) [reply src/test/java/org/jacp/repository/QuestionRepositoryTest.java:40]: This scenario includes checking for an invalid problem value

Comment (pelmegov) [reply src/test/java/org/jacp/repository/QuestionRepositoryTest.java:40]: Got it, it's potentially not required if you already checked correct value of problem, it looks like just test for test. Such scenarious should be avoided when possible.

Comment (saffchen) [reply src/test/java/org/jacp/repository/QuestionRepositoryTest.java]: Ok
Fixed

**Review by pelmegov — APPROVED** [review]

**Review by freesky15 — APPROVED** [review]

**Pull request merged**